notebooks/monitoring.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+echo ==================================
+echo =========== MONITORING ===========
+echo ==================================
+echo --- General Information ---
+# number of cores
+echo \#CPU: $(nproc)
+
+# multiply by 10^-6 to convert KB to GB
+echo Total Memory: $(cat /proc/meminfo | grep MemTotal | awk 'BEGIN { FS=" " } ; { print $2/1000000 }')G
+echo Total Disk space: $(df -h | grep cromwell_root | awk '{ print $2}')
```

Comment:
This could call the `get_disk_usage` function, once it exists.

Reply:
prints different field and is 1 line so no need

Reply:
Talked in person, but for the digital record: functions are good for giving descriptive names to things as well as for code reuse. Calling something like `get_cromwell_disk_usage` is much clearer than calling `df -h | grep cromwell_root`.